ipaserver/install/httpinstance.py

```diff
@@ -314,9 +315,8 @@ def create_cert_db(self):
 
         # Create the password file for this db
         password = ipautil.ipa_generate_password()
-        f = os.open(pwd_file, os.O_CREAT | os.O_RDWR)
-        os.write(f, password)
-        os.close(f)
+        with io.open(pwd_file, 'w') as f:
```

Comment:
'wt' is an uncommon way to spell out write in text mode. 'w' is the canonical way.